examples/plotting/file/color_data_map.py

```diff
@@ -1,3 +1,12 @@
+''' A color mapping plot with color spectrum scale. The example plots demonstrates
+log mapping and linear mapping with different color palette.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure.circle, bokeh.models.ColumnDataSource, bokeh.models.annotations.ColorBar, bokeh.models.mappers.LinearColorMapper, bokeh.models.mappers.LogColorMapper, # noqa: E501
```

Comment:
```suggestion
    :apis: bokeh.plotting.Figure.circle, bokeh.models.ColumnDataSource, bokeh.models.annotations.ColorBar, bokeh.models.mappers.LinearColorMapper, bokeh.models.mappers.LogColorMapper # noqa: E501
```
remove trailing comma